tests/WindowFixture.hpp

```diff
@@ -0,0 +1,33 @@
+# pragma once
+
+# include <gtest/gtest.h>
+
+# include <MLX42/MLX42.h>
+
+class Window : public ::testing::Test
+{
+protected:
+	mlx_t* mlx = NULL;
+
+	static constexpr const char *name = "MLX42";
```

Comment:
Uber cringe detected: `*` not with the type. 🤣

Reply:
okay okay okay
i will change my style